pageserver/src/layered_repository/inmemory_layer.rs

```diff
@@ -129,6 +129,11 @@ impl Layer for InMemoryLayer {
     }
 
     fn get_end_lsn(&self) -> Lsn {
+        if let Some(end_lsn) = self.end_lsn {
+            // TODO is this correct?
```

Comment:
@hlinnaka is this appropriate?

Reply:
I think we should just return the `end_lsn` as is and implement bound "exclusiveness" on a caller's side by using strict comparison operators where necessary.